testing/bdd/features/timing/delay.feature

```diff
@@ -68,13 +68,16 @@ Feature: Delay of a document sequence
     | 00:00:05   |              |           |          |          |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
```

Comment:
Yes, sorry for the confusion, I'm not sure what you discussed with @kozmaz87 exactly, but the behaviour of the `RetimingDelayNode` is defined by the change it makes between inputs and outputs, so that's what the BDD should check, rather than (or as well as) the internal computed values.